nixos/modules/config/console.nix

```
@@ -8,14 +8,22 @@ let
 
   makeColor = i: concatMapStringsSep "," (x: "0x" + substring (2*i) 2 x);
 
-  isUnicode = hasSuffix "UTF-8" (toUpper config.i18n.defaultLocale);
+  isUnicode = '' \
+    LOCALE_ARCHIVE=${pkgs.glibcLocales}/lib/locale/locale-archive \
```

Wouldn't it be better to use the locale archive generated by the `i18n` module?

```suggestion
    LOCALE_ARCHIVE=${config.i18n.glibcLocales}/lib/locale/locale-archive \
```

---

Done.